### PR TITLE
Remove old version variants in instance creation

### PR DESCRIPTION
**Pull request by ColonelGerdauf**
This should make things more simple to view and set. The existence of `old_*` is frankly unnecessary. There is no "newer" alpha or beta, and in the case of snapshot, the distinction is insignificant to the end users.

More to the point, the current search regex includes `old_snapshot` when looking up regular snapshots, which makes the thing pointless.
~blocked by: https://github.com/PrismLauncher/meta/pull/43~

### CODE REVIEW

**Comment by TheKodeToad** [comment]
It doesn't make a lot of sense to change the old_alpha and old_beta filters because the version types alpha and beta don't exist. Also, the distinction of snapshot and old_snapshot exists in the type column so I don't see why it shouldn't in the filter.

**Comment by TheKodeToad** [comment]
Hm, it might make sense to get rid of old_snapshot if it's done in meta too though.
It doesn't make sense a lot of the time...
image

**Comment by ColonelGerdauf** [comment]
I am not sure how important or useful that is for the end users. If there is no regular alpha or beta, then old_alpha, old_beta feels quite silly. Same for snapshot; what is the benefit of putting it in front of user-facing data?

I am not denying the use on the back end; it may very well be needed to get and aggregate the list items in the first place.

The point of this PR is for the user-side interfacing.

**Comment by TheKodeToad** [comment]
On second thoughts this is probably a good change, meta should be changed though. I was confused because you didn't mention anything about it.

Feel free to open a PR in https://github.com/PrismLauncher/meta otherwise somebody else can before this is merged!

**Comment by ColonelGerdauf** [comment]
And the preliminary work is done in Meta... I hope

**Review by Trial97 — APPROVED** [review]

**Review by Trial97 — APPROVED** [review]

**Comment by Trial97** [comment]
This is no longer blocked by meta and can be reviewed

**Review by Scrumplex — APPROVED** [review]

**Pull request merged**
